lib/python/script/utils.py

```diff
@@ -467,3 +471,104 @@ def clock():
     if sys.version_info > (3,2):
         return time.perf_counter()
     return time.clock()
+
+
+def legalize_vector_name(name):
+    if not name:
+        raise ValueError("name cannot be empty")
+    if re.match("[^A-Za-z0-9]", name[0]):
+        name[0] = "_"
```

Comment:
Are you sure this works ? I get "TypeError: 'str' object does not support item assignment".

Reply:
> Are you sure this works ? I get "TypeError: 'str' object does not support item assignment".

Same error here (for all names not starting with a special character).

Reply:
Thanks for catching that! I have realized this would be a bad fix anyway. There is something like this in some vector import code and there columns like 1000 and 2000 (years, ZIP codes, ...) will result in the same name, so prefixing is needed.